### PR TITLE
KAZOO-5597 sort dispatcher groups and display flags

### PR DESCRIPTION
**Pull request by lazedo**

### CODE REVIEW

**Comment by lazedo** [comment]
kamailio script https://github.com/2600hz/kazoo-configs-kamailio/pull/15

File: core/kazoo_globals/src/kz_nodes.erl [hunk 1]
```diff
@@ -339,7 +339,11 @@ maybe_print_roles(Roles) ->
 -spec print_role({ne_binary(), kz_json:object()}) -> 'ok'.
 print_role({<<"Dispatcher">>, Data}) ->
     Groups = kz_json:get_json_value(<<"Groups">>, Data, kz_json:new()),
-    kz_json:foreach(fun print_dispatcher/1, Groups);
+    Keys = lists:sort(kz_json:get_keys(Groups)),
+    lists:foreach(fun(Group) ->
+                          GData = kz_json:get_json_value(Group, Groups),
+                          print_dispatcher({Group, GData})
+                  end, Keys);
```
Comment (fenollp) [inline core/kazoo_globals/src/kz_nodes.erl]: What changed here? Is there something wrong with `kz_json:foreach/2`?

Comment (fenollp) [inline core/kazoo_globals/src/kz_nodes.erl]: Hm it's for get_json_value... How about calling is_json_object in print_dispatcher/1 instead?

Comment (lazedo) [inline core/kazoo_globals/src/kz_nodes.erl]: what's wrong with this ?

Comment (lazedo) [inline core/kazoo_globals/src/kz_nodes.erl]: the keys for the dispatcher are not sorted, we get those, sort them and the elements by that order

**Pull request merged**